pyproject.toml

```diff
@@ -25,3 +25,12 @@ docstring-convention = "google"
 copyright-check = "True"
 copyright-author = "Canonical Ltd."
 copyright-regexp = "Copyright\\s\\d{4}([-,]\\d{4})*\\s+%(author)s"
+
+[tool.pytest.ini_options]
+markers = [
+    "v1dot4: bundle 1.4 tests",
+    "v1dot6: bundle 1.6 tests",
+    "full_bundle_shared: tests that are shared in all versions of full bundle",
+    "lite_bundle_shared: tests that are shared in all versions of lite bundle"
```

Comment:
Is `_shared` needed here?  I think you could get away with `full_bundle` or even `full`

Reply:
Oh also, however we mark the different releases (full_bundle_shared, lite_bundle, etc), let's mark it with the same name as we use in the release directories themselves (eg, in `releases/x.y/THIS`).  It'll make writing CI easier (no need to map `release=kubeflow` means `mark=full_bundle`, etc).  Atm those folders are `kubeflow` and `kubeflow-lite` (the names match the bundle's actual name in charmhub), but if you have better ideas they'd be appreciated.

This might be a comment more directed at the `tox.ini` envs rather than pytest marks, since that is our entrypoint.  But if possible, keeping everything consistent will make it easier to use